### PR TITLE
i18n: German translation switches 'conversation' and 'discussion'

### LINKED ISSUE #31587: Translations: steamline discussions to either "Unterhaltung" or "Diskussion"
### Description:

The german translations use both *Diskussion* and *Unterhaltung* to refer to the discussions feature. This should be steamlined to either one of those. Both work out from a translation point of view.

### Steps to reproduce:

grafik

grafik

grafik

### Expected behavior:

Either *Diskussion* or *Unterhaltung* should be used in the translation, not both of them.

### Actual behavior:

See screenshots. Might not contain all locations where the translations appear.

### Server Setup Information:

- Version of Rocket.Chat Server: 6.5.2
- Operating System: Ubuntu 22.04 LTS
- Deployment Method: Docker
- Number of Running Instances: 
- DB Replicaset Oplog: 
- NodeJS Version: 
- MongoDB Version:

### Client Setup Information

- Desktop App or Browser Version:
- Operating System:

### PR DESCRIPTION
**Pull request by hardikbhatia777**
## Proposed changes (including videos or screenshots)

German translation files don't differentiate between 'Unterhaltung' (meaning entertainment / conversation) and 'Diskussion' (Discussion). This creates inconsistencies in labels and tool tips across the UI and should be streamlined.

I've replaced all appropriate occurances of the same to match the exact translation of English (and verified them using Google Translate). 

If I replaced something that's inappropriate, please let me know :) I'll change it right away.
## Issue(s)

Closes #31587

### CODE REVIEW

File: apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json [hunk 1]
```diff
@@ -3445,8 +3445,8 @@
   "No_channels_yet": "bisher nirgendwo dabei",
   "No_data_found": "Keine Daten gefunden",
   "No_direct_messages_yet": "Sie haben bisher keine Konversationen gestartet",
-  "No_Discussions_found": "Keine Unterhaltungen gefunden",
-  "No_discussions_yet": "Keine Unterhaltungen vorhanden",
+  "No_Discussions_found": "Keine Diskussion gefunden",
```
Comment (david-uhlig) [inline apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json:3448]: This needs to be plural: Diskussionen

File: apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json [hunk 2]
```diff
@@ -3445,8 +3445,8 @@
...
+  "No_discussions_yet": "Keine Diskussion vorhanden",
```
Comment (david-uhlig) [inline apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json:3449]: This needs to be plural: Diskussionen

File: apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json [hunk 3]
```diff
@@ -4771,7 +4771,7 @@
   "Total": "Gesamt",
   "Total_abandoned_chats": "Total aufgegebene Chats",
   "Total_conversations": "Konversationen insgesamt",
-  "Total_Discussions": "Anzahl aller Unterhaltungen",
+  "Total_Discussions": "Anzahl aller Diskussions",
```
Comment (david-uhlig) [inline apps/meteor/packages/rocketchat-i18n/i18n/de.i18n.json:4774]: This needs to be plural: Diskussionen

**Review by david-uhlig — CHANGES_REQUESTED** [review]
Thank you again for providing a quick fix @hardikbhatia777! 

I marked three occurences where we need to use the plural. Apart from these the changes look fine to me. Could you also please change the de-IN and de-AT translation files. They have the same inconsistencies. Appreciated!

**Review by david-uhlig — APPROVED** [review]
Looks good to me! There seems to be a fallback to de.i18n.json for the missing de-AT.i18n.json lines, so this works out fine. Well done.

**Review by rique223 — APPROVED** [review]
Great work folks. Looks good to me.

**Pull request merged**